dts/bindings/power-domain/power-domain-gpio.yaml

```diff
@@ -28,3 +28,7 @@ properties:
     type: int
     default: 0
     description: Off delay time, in microseconds
+
+  boot-on:
+    type: boolean
+    description: Sets whether the domain to power on at boot
```

Comment:
My preference is for something closer to "If set the regulator is automatically powered on at boot time". Minor suggestion.

Reply:
sure, updated